components/authz-service/server/server.go

```diff
@@ -71,7 +71,9 @@ func NewGRPCServer(ctx context.Context,
 		return nil, errors.Wrap(err, "could not initialize v1 server")
 	}
 
-	v2PolServer, err := v2.NewPostgresPolicyServer(ctx, l, e, migrationsConfig,
+	// TODO (tc) Refactor how singletons are shared between GRPC servers. Should we also
```

Comment:
💭 This is what the _rearchitect authz-service_ card is about, too. Suggestions welcome 😉